src/test/java/com/google/javascript/clutz/partial/import_without_reexport.js

```diff
@@ -1,12 +1,9 @@
 //!! Tests that the alias map doesn't force aliases for things that aren't
 //!! actually aliases
-//!! TODO(lucassloan): actually parse the `exports = foo` or `exports.foo = foo`
-//!! statements, to actually solve this problem
 goog.module('import.without.reexport');
 
 const {Class} = goog.require('original.module');
 
-/** @const {type.not.present} */
-const x = null;
+const x = unknown.function(Class);
```

Comment:
OOC, why change this test? Maybe it wasn't a good enough case for the real internal google error, but then keep the old assertion and add a new one

`exports.x = x` and `exports.y = y`.

Reply:
Sure.